lms/templates/student_account/hinted_login.underscore

```diff
@@ -0,0 +1,24 @@
+<div class="hint-wrapper">
+    <div class="section-title lines">
+        <h2>
+            <span class="text"><%- gettext("Sign in") %></span>
+        </h2>
+    </div>
+
+    <p><%- _.sprintf( gettext("Would you like to sign in using your %(providerName)s credentials?"), { providerName: hintedProvider.name } ) %></p>
+
+    <button class="action action-primary action-update proceed-button button-<%- hintedProvider.id %> hinted-login-<%- hintedProvider.id %>">
+        <div class="icon fa <%- hintedProvider.iconClass %>" aria-hidden="true"></div>
```

Comment:
@talbs It's fixed on the other PR. I've added "Dummy" to that sandbox so you can see: http://sandbox4.opencraft.com/login

This PR doesn't make any changes to that page, it just adds the new "hinted" page that asks about one specific provider.